cmd/inspect.go

```diff
@@ -43,9 +47,10 @@ func newInspectCmd() *cobra.Command {
 				arg = args[0]
 			}
 
-			return impl.InspectCmd(globalClient, arg)
+			return impl.InspectCmd(globalClient, arg, o)
 		},
 	}
+	cmd.Flags().BoolVarP(&o.ShowQueryModifyPermission, "show-permission", "p", false, "Show query modify permission")
```

Comment:
@litencatt 
I prefer to use local variable for flag options as `add` command.

Like this:
```go
			var o impl.Options
			if showPermission, err := cmd.Flags().GetBool("show-permission"); err != nil {
				return err
			} else {
				o.ShowQueryModifyPermission = showPermission
			}

			return impl.InspectCmd(globalClient, arg, o)
		},
	}
	cmd.Flags().BoolP("show-permission", "p", false, "Show query modify permission")
```

Reply:
@snowhork 
Thanks. I fixed it.